test/fixtures/render/object-array-string.json

```diff
@@ -184,7 +184,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"a\": [\n    \"b\",\n    \"c\"\n  ]\n}"
+                              "content": "{\n  \"a\": [\n    \"\",\n    \"b\",\n    \"c\"\n  ]\n}"
```

Comment:
This introduces an empty string into the rendered array which I do not believe should be there. This is being done based on the type-hint of the array (`array[string]`).